### PR TITLE
[Skia] Add SkRuntimeEffect Fuzzer

### PR DESCRIPTION
**Pull request by werdnaH**
@kjlubick @jonathanmetzman

### CODE REVIEW

File: projects/skia/build.sh [hunk 1]
```diff
@@ -222,3 +222,6 @@ cp ./svg_dom_seed_corpus.zip $OUT/svg_dom_seed_corpus.zip
 
 cp out/Fuzz/api_svg_canvas $OUT/api_svg_canvas
 cp ./canvas_seed_corpus.zip $OUT/api_svg_canvas_corpus.zip
+
+cp out/Fuzz/skruntimeeffect $OUT/skruntimeeffect
+cp ./sksl_with_256_padding_seed_corpus.zip $OUT/sksl_with_256_padding_seed_corpus.zip
```
Comment (kjlubick) [inline projects/skia/build.sh:227]: This will not work - the name must match the name of the fuzzer. skruntimeeffect_seed_corpus.zip

Comment (kjlubick) [reply projects/skia/build.sh:227]: That is, the name in the $OUT directory must match.

File: projects/skia/build.sh [hunk 2]
```diff
@@ -224,4 +224,4 @@ cp out/Fuzz/api_svg_canvas $OUT/api_svg_canvas
 cp ./canvas_seed_corpus.zip $OUT/api_svg_canvas_corpus.zip
```
Comment (kjlubick) [inline projects/skia/build.sh:224]: api_svg_canvas_seed_corpus.zip

File: projects/skia/build.sh [hunk 3]
```diff
@@ -224,4 +224,4 @@ cp out/Fuzz/api_svg_canvas $OUT/api_svg_canvas
...
 
 cp out/Fuzz/skruntimeeffect $OUT/skruntimeeffect
-cp ./sksl_with_256_padding_seed_corpus.zip $OUT/sksl_with_256_padding_seed_corpus.zip
+cp ./sksl_with_256_padding_seed_corpus.zip $OUT/skruntimeeffect.zip
```
Comment (kjlubick) [inline projects/skia/build.sh:227]: skruntimeeffect_seed_corpus.zip

File: projects/skia/build.sh (hunk 1 above)
Comment (werdnaH) [reply projects/skia/build.sh]: I have fixed issue and build and run the code, please take another look.

**Pull request merged**